webpack.config.js

```diff
@@ -0,0 +1,33 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
```

Comment:
- [ ] As requested by the previous reviewer, I still can't open your app in the browser even after running `npm install`, `npm run build`, and `npm start`. Please configure webpack properly so that we can run the application and see the result in the browser without any errors 🙃 👇🏻

- In the Browser:

![webpa](https://user-images.githubusercontent.com/79058364/171622257-31aed7da-6933-4013-9444-22069eb3c041.png)


- In the Console:

![wekpack](https://user-images.githubusercontent.com/79058364/171622383-d1cc77d5-7bc2-4eda-95ef-4fc15e124dd4.png)